CONTRIBUTING.md

```diff
@@ -9,6 +9,35 @@ We welcome contributions in the form of pull requests and issues.
 
 Note that this codebase isn't yet extensively documented. If you get stuck, please ask for help [on Discord](https://docs.grit.io/discord).
 
+## Development Setup
+
+A high level overview of tools you need to have installed:
+
+* Rust compiler. You'll need [`rustc`](https://rustup.rs/) v1.74 or newer.
```

Comment:
This is redundant from the `## development tools` section below - please consolidate.

Reply:
Done. Please see [dbe396f](https://github.com/ayewo/gritql/commit/dbe396f4f73c18e616da0fb361d38079cb3bad53).